test/vmtester/vmtester.cpp

```diff
@@ -35,6 +35,38 @@ evmc_instance* get_vm_instance()
     return vm.get();
 }
 
+std::vector<std::string> get_vm_names(const fs::path path)
```

Comment:
Why not make this into a helper in the library (like instructions)? It has to have the filename as an input string though, so in the `vmtester` you could have this extra function to deal with `fs::path`.

Reply:
Maybe I will do it later to use in aleth. But not useful for geth anyway.